data/remnant/remnant missions.txt

```diff
@@ -2208,3 +2208,633 @@ mission "Remnant: Face to Maw 2B"
+		object
+			distance 439
+			period 143
+			sprite planet/desert7
+		object
+			distance 814
+			period 362
+			sprite planet/fog0
+			object
+				distance 202
+				period 21
+				sprite planet/dust4
+		object
+			period 19
+			sprite star/m4
+		pos 297 269
+	unvisit Levana
+	unvisit Caeculus
+	unvisit Prosa
+	unvisit Egeria
+	system Levana
+		add object wormhole_barren
+			distance 2313
+			period 389
+			sprite planet/wormhole-red
+	system Caeculus
+		add object wormhole_barren2
+			distance 2400
+			period 391
+			sprite planet/wormhole-red
+	system Prosa
+		add object wormhole_barren2
+			distance 2313
+			period 389
+			sprite planet/wormhole-red
+	system Egeria
+		add object wormhole_barren
+			distance 2313
+			period 389
+			sprite planet/wormhole-red
+		
+
+
+mission "Remnant: Cognizance 12"
+	name "Investigate strange sensor readings"
+	description "Long range sensors have picked up strange gravitational waves. Go investigate."
+	landing
+	source "Viminal"
+	waypoint
+		system Postverta
+	stopover
+		planet "Ssil Vida"
+	to offer
+		has "Remnant: Cognizance 11: done"
+	on offer
+		conversation
+			`You land by the starport and is immediately accosted by a Remnant who informs you that Prefect Chilia is on his personal warship before escorting you to a nearby starling. Inside you find Chilia and several other Remnants clustered around a holographic display and some display panels. "Ah, Captain. Join us, please." Chilia gestures for you to join them around the displays. As you approach, you note that the holographic display shows the Ember Waste, although there are a few details that do not match normal star maps.`
+			`	Chilia notices you eyeing the deviations. "Perceptive, captain. Our sensors have reported some unusual energy signatures coming from this location. Scouts in the area says they are picking up signs of a new system. There were no immediately apparent hyperlinks or wormholes that connect to it, but there are still unusual energy fluctuations in the area. Unusual even for the Ember Waste." He looks up at you. "We are hoping that you might be able to figure out a way in."`
+			choice
+				`	"I could see if I can reach it with a Jump Drive."`
+				`	"I will see if I can find another way."`
+			`	"Thank you, Captain <first>. We have some ships that can follow, and there are other scouts already in the area that should be reporting soon. Given your experience with the wider galaxy, though, we would like you to lead the way."`
+				accept
+	on enter "Postverta"
+		dialog
+			`As the sensors re-establish their depiction of the system around you, an alien station immiediately becomes apparent. It appears to be old and mostly inactive, but your sensors do pick up ocasional bursts of energy. Oddly, your sensors are reporting faint signs of life almost everywhere.`
+	on stopover
+		dialog
+			`The station is surprisingly still active, despite its apparent age. You poke around a bit, but its mysteries remain just that.`
```

Comment:
I feel like we need a bit more description than this. :) What's it like inside the alien space station?

Reply:
Significantly expanded the detail for this mission.